.github/workflows/node-upgrade.yml

```diff
@@ -0,0 +1,92 @@
+name: Auto Update Node Version
+
+on:
+  schedule:
+    - cron: "0 6 * * 1" # Weekly, every Monday
+  workflow_dispatch:
+
+jobs:
+  update-node:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - name: Get latest Node versions
+        id: node-versions
+        run: |
+          # Get latest versions from the versions manifest
+          VERSIONS_JSON=$(curl -s https://raw.githubusercontent.com/actions/node-versions/main/versions-manifest.json)
```

Comment:
I think we should use `https://github.com/actions/alpine_nodejs/releases` and `https://github.com/nodejs/node/releases` to help decide which version is good for us.